test/conformance/conformance_global_control.cpp

```diff
@@ -252,7 +356,7 @@ TEST_CASE("terminate_on_exception: enabled") {
             CHECK(!terminate_handler_called);
             terminate_handler_called = true;
             std::longjmp(buffer, 1);
-        });
+            });
```

Comment:
I would personally prefer the previous variant? as more logically consistent

Reply:
It seems as an occasional change. 
```suggestion
        });
```